client/my-sites/domains/style.scss

```diff
@@ -1,5 +1,6 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
+@import "@automattic/typography/styles/variables";
```

Comment:
Is this necessary? Isn't it already imported here:

https://github.com/Automattic/wp-calypso/blob/b7a4a072ae2a13785fb6ccbf9c34ed8e65739705/client/assets/stylesheets/shared/_typography.scss#L2

Reply:
Yeah, it's not necessary, [fixed here](https://github.com/Automattic/wp-calypso/pull/73017/commits/ccaaa9cb70837c2e4fd33ab7f6b5656062617386).